elements/lisk-transaction-pool/test/unit/transaction_list.spec.ts

```diff
@@ -541,7 +561,7 @@ describe('TransactionList class', () => {
 			});
 		});
 
-		describe('when there are only unprocessable transactions and all nonce are partially continuous', () => {
+		describe('when there are only unprocessable transactions and all nonce are in sequence', () => {
```

Comment:
```suggestion
		describe('when there are only unprocessable transactions and all nonces are in sequence', () => {
```

Reply:
the plural of nonce is also missing in the previous descriptions of `getPromotable` tests.